packages/client/lib/sync/fetcher/fetcher.ts

```diff
@@ -312,7 +312,7 @@ export abstract class Fetcher<JobTask, JobResult, StorageItem> extends Readable
     this.nextTasks()
     const job = this.in.peek()
     if (!job) {
-      if (this.finished !== this.total) {
+      if (this.finished >= this.total) {
```

Comment:
when can finished be > total? if this is happening indicates a bug somewhere

Reply:
Yeah, I had a feeling that it would be more of a bug than a feature. It happens with the storage fetcher. I'll take another look at this.

Reply:
ok i have found the source of bug :sweat_smile: , will fix and update PR